borg/archiver.py

```diff
@@ -1730,12 +1730,12 @@ def main():  # pragma: no cover
     except RemoteRepository.RPCError as e:
         msg = '%s\n%s' % (str(e), sysinfo())
         exit_code = EXIT_ERROR
+    except (KeyboardInterrupt, BrokenPipeError):
+        sys.stderr.close()
+        exit_code = EXIT_SUCCESS
```

Comment:
@ThomasWaldmann My logic here is that `borg` didn't really have any errors. The user just decided that he doesn't want run the command further. However, I am okay with non-zero return code, too, if you think it's better that way.

@enkore It is to avoid any further writes. Otherwise TextIOWrapper will try to use it and get an error after this exception handling.